packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -22,6 +22,22 @@ const postcssColorModFunction = require('postcss-color-mod-function');
 const FinalStatsWriterPlugin = require('../webpack-plugins/final-stats-writer-plugin');
 const browserslist = require('./browserslist');
 
+const uglifyConfig = {
+  uglifyOptions: {
+    warnings: false,
+    parse: {},
+    compress: {},
+    mangle: true,
+    output: null,
+    toplevel: false,
+    nameCache: null,
+    ie8: false,
+    // https://github.com/mishoo/UglifyJS2/issues/1609
+    collapse_vars: false,
```

Comment:
this fixing a problem in a few apps

Reply:
So we're adding back what we had before? 🤔

Reply:
sort of, what we had before was for a different uglifer